scripts/circleci/setup-ssm-for-back-migration-job.sh

```diff
@@ -6,11 +6,14 @@
   "AWS_ACCESS_KEY_ID" \
   "AWS_SECRET_ACCESS_KEY"
 
+# TODO: we may still need this
 echo "setting migrate flag to false"
 aws ssm put-parameter --region us-east-1 --name "/DAWSON/${ENV}/migrate" --value "false" --type "String" --overwrite
 
+# TODO: I'm not sure we need this anymore
 echo "setting source-table-version to alpha"
 aws ssm put-parameter --region us-east-1 --name "/DAWSON/${ENV}/source-table-version" --value "alpha" --type "String" --overwrite
 
+# TODO: I'm not sure we need this anymore
 echo "setting destination-table-version to alpha"
 aws ssm put-parameter --region us-east-1 --name "/DAWSON/${ENV}/destination-table-version" --value "alpha" --type "String" --overwrite
```

Comment:
We can only remove this if there's no code anywhere that changes the version to `beta` for any reason. Because you're right - `test` should already be on alpha (considering we run this script once a month and nothing switches from alpha to beta anymore) by now.